### ISSUE TITLE
Don't immediately redirect non-linux to elementary.io

### ISSUE THREAD

**Issue by danirabbit** [elementary/appstream-redirect#9]
Seems like users are reporting being confused about this redirection and unsure why it happened. It would probably be better to offer some kind of explanation and give a button to go to the homepage

**Comment by cassidyjames** [comment]
At the least, some sort of generic "Get this app on elementary OS" landing page would work. #10 would also solve this, but will be significantly more work.

**Comment by cassidyjames** [comment]
Something like this:

># Get this app on elementary OS
>
>This app is available in AppCenter, the open, pay-what-you-want app store for elementary OS. Download elementary OS to use AppCenter and install this app.
>
>**`Download elementary OS`**

...that seems a bit wordy though. :stuck_out_tongue:
